source/UIAHandler/__init__.py

```diff
@@ -1170,6 +1167,13 @@ def _isUIAWindowHelper(self, hwnd: int, isDebug=False) -> bool:  # noqa: C901
 					if isDebug:
 						log.debug("Windows console treated as non-UIA")
 					return False
+			elif windowClass == "SysListView32":
+				# #15283: SysListView32 controls in Windows Forms have a native UIA implementation
+				# and lack a MSAA implementation.
+				# We need to rely on UIA for these controls, as otherwise parent/child navigation is broken.
+				# For other instances however, even when the control advertises a native UIA implementation,
+				# the implementation is likely to be incomplete and MSAA should be prefered.
+				return utils._isFrameworkIdWinForm(hwnd)

```

Comment:
I would like to see some logging either in or around this call, especially because it fetches a UIA element in order to check. Logging to say that it is going to check, and also logging the answer, as this is what then causes NvDA to decide if the window is native or not. See other examples of logging in this function.